pkg/nodeconfig/payload/payload.go

```diff
@@ -188,13 +224,13 @@ func PopulateNetworkConfScript(clusterCIDR, hnsNetworkName, hnsPSModulePath, cni
 
 // generateNetworkConfigScript generates the contents of the .ps1 file responsible for CNI configuration
 func generateNetworkConfigScript(clusterCIDR, hnsNetworkName, hnsPSModulePath,
-	cniConfigPath string) (string, error) {
+	cniDir string) (string, error) {
```

Comment:
we should also rename the parameter in the parent func `PopulateNetworkConfScript()`